### PR TITLE
Fix 26

### LINKED ISSUE #26: Make our `error.tsx` page consistent
## Description 📹

[The error.tsx page 
](https://github.com/lfglabs-dev/affiliate.starknet.id/blob/testnet/pages/error.tsx) used for asking the user to connect isn't the best right now and we want to add an image to it to illustrate the problem better. Just add [this image](https://github.com/lfglabs-dev/app.starknet.id/blob/testnet/public/visuals/errorIllu.webp) as webp to the component (and in the right folder).

## Proposed Actions 🛠️

Here’s a checklist of actions to follow for resolving this issue:

1. **Fork and Create Branch**:  
   Fork the repository and create a new branch using the issue number:
   ```bash
   git checkout -b fix-[issue-number]
   ```

2. **Implement Changes**:  
	[Insert Code snippet if needed]

3. **Run Tests and Commit Changes**:  
   Make sure your changes don't break existing functionality and commit with a clear message:
   ```bash
   git commit -m "Fix: [Short description of the fix]"
   ```

## Required 📋

To keep our workflow smooth, please make sure you follow these guidelines:

- **Assignment**: Don't create a pull request if you weren’t assigned to this issue.
- **Timeframe**: Complete the task within **3 business days**.
- **Closing the Issue**: In your PR description, close the issue by writing `Close #[issue_id]`.
- **Review Process**:
  - Once you've submitted your PR, change the label to **"ready for review"**.
  - If changes are requested, address them and then update the label back to **"ready for review"** once done.
- **Testing**: Test your PR locally before pushing, and verify that tests and build are working after pushing.

Thank you for your contribution 🙏

⚠️ WARNING: Failure to follow the requirements above may result in being added to the OnlyDust blacklist, affecting your ability to receive future rewards.

### PR DESCRIPTION
**Pull request by Michaelkingsdev**
Fixes #26 

# Before
image
# After
image

### CODE REVIEW

**Comment by Michaelkingsdev** [comment]
@fricoben 
please review and give corrections. Thanks

**Review by fricoben — CHANGES_REQUESTED** [review]
- [ ] Background color is not the same 
- [ ] The image introduces a scroll on laptop (we don't want that). 
- [ ] Section isn't centered

Please take example on this page https://app.starknet.id/qsd

**Comment by Michaelkingsdev** [comment]
> * [ ]  Background color is not the same
> * [ ]  The image introduces a scroll on laptop (we don't want that).
> * [ ]  Section isn't centered
> 
> Please take example on this page https://app.starknet.id/qsd

Alright. Thanks

**Comment by Michaelkingsdev** [comment]
> > * [ ]   Background color is not the same
> > * [ ]   The image introduces a scroll on laptop (we don't want that).
> > * [ ]   Section isn't centered
> > 
> > Please take example on this page https://app.starknet.id/qsd
> 
> Alright. Thanks

@fricoben 
This link:  https://app.starknet.id/qsd is not accessible.

**Comment by fricoben** [comment]
> > > * [ ]    Background color is not the same
> > > * [ ]    The image introduces a scroll on laptop (we don't want that).
> > > * [ ]    Section isn't centered
> > > 
> > > Please take example on this page https://app.starknet.id/qsd
> > 
> > 
> > Alright. Thanks
> 
> @fricoben This link: https://app.starknet.id/qsd is not accessible.

1. It's a 404 page to give you example
2. yes it's accessible

**Comment by Michaelkingsdev** [comment]
> > > > * [ ]     Background color is not the same
> > > > * [ ]     The image introduces a scroll on laptop (we don't want that).
> > > > * [ ]     Section isn't centered
> > > > 
> > > > Please take example on this page https://app.starknet.id/qsd
> > > 
> > > 
> > > Alright. Thanks
> > 
> > 
> > @fricoben This link: https://app.starknet.id/qsd is not accessible.
> 
> 1. It's a 404 page to give you example
> 2. yes it's eccessible

image

**Comment by fricoben** [comment]
> > > > > * [ ]      Background color is not the same
> > > > > * [ ]      The image introduces a scroll on laptop (we don't want that).
> > > > > * [ ]      Section isn't centered
> > > > > 
> > > > > Please take example on this page https://app.starknet.id/qsd
> > > > 
> > > > 
> > > > Alright. Thanks
> > > 
> > > 
> > > @fricoben This link: https://app.starknet.id/qsd is not accessible.
> > 
> > 
> > 
> > 1. It's a 404 page to give you example
> > 2. yes it's eccessible
> 
> image

You have a problem on your laptop or network it seems, please use another device/network

**Comment by Michaelkingsdev** [comment]
> > > > > > * [ ]       Background color is not the same
> > > > > > * [ ]       The image introduces a scroll on laptop (we don't want that).
> > > > > > * [ ]       Section isn't centered
> > > > > > 
> > > > > > Please take example on this page https://app.starknet.id/qsd
> > > > > 
> > > > > 
> > > > > Alright. Thanks
> > > > 
> > > > 
> > > > @fricoben This link: https://app.starknet.id/qsd is not accessible.
> > > 
> > > 
> > > 
> > > 1. It's a 404 page to give you example
> > > 2. yes it's eccessible
> > 
> > 
> > image
> 
> You have a problem on your laptop or network it seems, please use another device/network

Hello @fricoben 
Can you send a screenshot of that page. I think its better that way. i cant access it from my end.

**Comment by Michaelkingsdev** [comment]
@fricoben  Done
image

**Review by fricoben — APPROVED** [review]
lgtm

File: pages/error.tsx [hunk 1]
```diff
@@ -2,18 +2,27 @@ import React, { FunctionComponent } from "react";
...
+    <div className={`${styles.container}`}>
       <div className={errorStyle.main}>
-        <h2>Oh no... access error!</h2>
-        <p className="text-small mt-8">
-          To access to your stark affiliate space you need to connect to a
-          starknet wallet
-        </p>
+        <div className="flex flex-col items-center justify-center">
+          <img 
+            src="/visuals/errorillu.webp"
```
Comment (fricoben) [inline pages/error.tsx]: NO `errorIllu.web` don't forget the uppercase

**Review by fricoben — CHANGES_REQUESTED** [review]
Don't forget the uppercase ! Test the build after sending PR

**Comment by Michaelkingsdev** [comment]
@fricoben 
Done Sir

**Review by fricoben — CHANGES_REQUESTED** [review]
Capture d’écran 2024-11-27 à 09 34 04

Your PR completely destroys the rest of the website here, pls fix and TEST before sending your PR

**Comment by fricoben** [comment]
@Michaelkingsdev PLS fix

**Comment by Michaelkingsdev** [comment]
> @Michaelkingsdev PLS fix

Alright sir

**Comment by Michaelkingsdev** [comment]
@fricoben Are you sure this is not how the page looks?
I recloned the repo and the page still looks the same. Should i fix it by the way. 
@fricoben I am sorry if i am stressing you.
image

This is the mobile view. Note this is a fresh clone of the `testnet` branch. 
Should i fix it?

[Watch here](https://www.webmobilefirst.com/en/screencasts/lk5rnhjypacf7e/)

**Review by fricoben — APPROVED** [review]
lgtm

**Pull request merged**